packages/core-mobile/app/new/features/hardware/screens/KeystoneSignerScreen/index.tsx

```diff
@@ -0,0 +1,190 @@
+import { Button, Text, useTheme, View } from '@avalabs/k2-alpine'
+import React, { FC, useCallback, useEffect, useState } from 'react'
+import { withWalletConnectCache } from 'common/components/withWalletConnectCache'
+import { KeystoneSignerParams } from 'services/walletconnectv2/walletConnectCache/types'
+import { useNavigation } from 'expo-router'
+import { ScrollScreen } from 'common/components/ScrollScreen'
+import { UREncoder } from '@ngraveio/bc-ur'
+import { Space } from 'common/components/Space'
+import QRCode from 'react-native-qrcode-svg'
+import { Dimensions, BackHandler } from 'react-native'
+import { KeystoneQrScanner } from 'common/components/KeystoneQrScanner'
+import KeystoneLogoLight from 'assets/icons/keystone_logo_light.svg'
+import KeystoneLogoDark from 'assets/icons/keystone_logo_dark.svg'
+
```

Comment:
Could you also feature flag all the entry points for this integration (ui, redux listener) using this PostHog flag `keystone3-onboarding`? You can refer to [this commit](https://github.com/ava-labs/core-mobile/pull/2873/commits/542cf6743e6e2a087691ff2f3dce581539d2dda2#diff-d7768d6d58141e37870f484c66b00ef548228194119a931ff867a2b2eb326622) as an example.

<img width="521" height="248" alt="Screenshot 2025-07-16 at 12 12 43 PM" src="https://github.com/user-attachments/assets/6e0c8fae-2a46-494f-a384-211e7de19e2b" />